frontends/ol-test-utilities/src/querify.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * Returns a query string from an object.
+ *
+ * For example:
+ * ```ts
+ * querify(undefined) // ""
+ * querify({}) // ""
+ * querify({ a: 1 }) // "?a=1"
+ * querify({ a: 1, b: 2 }) // "?a=1&b=2"
+ * querify({ a: 1, b: [2, 3] }) // "?a=1&b=2&b=3"
+ * ```
+ *
+ * The query string is sorted by key.
+ */
+const queryify = (params: unknown, { explode = true } = {}) => {
```

Comment:
This function is moved from `api/src/test-utils/index.ts`. The only change was addition of `explode` option, since mitxonline does not explode query parameters.